install_config/cluster_metrics.adoc

```diff
@@ -125,6 +125,10 @@ If you would like to use xref:../install_config/persistent_storage/dynamically_p
 xref:../install_config/cluster_metrics.adoc#metrics-ansible-variables[variable]
 to `dynamic` in the inventory file.
 
+You can use {gluster-native} to enable `gluster-block` Storage
+as a backend for logging with Elasticsearch. For more information, see
```

Comment:
Looks like this is a copy/paste error from the Logging file. This is likely supposed to talk about Metrics.

Reply:
Thanks, @jarrpa. I get that fixed today.